spec/spec_helper.rb

```diff
@@ -38,15 +39,15 @@
 if defined? JRUBY_VERSION
   DB_URIS = {
     sqlite: 'jdbc:sqlite:',
-    postgres: 'jdbc:postgresql://localhost/rom_sql',
-    mysql: 'jdbc:mysql://localhost/rom_sql?user=root&sql_mode=STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION&useSSL=false',
+    postgres: ENV.fetch('POSTGRES_DSN', 'jdbc:postgresql://localhost/rom_sql'),
+    mysql: ENV.fetch('MYSQL_DSN', 'jdbc:mysql://localhost/rom_sql?user=root&sql_mode=STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION&useSSL=false'),
```

Comment:
@GustavoCaso That is exactly what is happening here. Use environment variable, and if not present, which is the default case, use "default" DSN.

Reply:
Awesome, sorry I was not very familiar with `fetch` and not using block